src/CppParser/Bootstrap/Bootstrap.cs

```diff
@@ -31,27 +31,30 @@ static string GetSourceDirectory(string dir)
         public void Setup(Driver driver)
         {
             var options = driver.Options;
-            options.LibraryName = "CppSharp";
+            options.GeneratorKind = GeneratorKind.Csharp;
             options.DryRun = true;
-            options.Headers.AddRange(new string[]
```

Comment:
Sorry, I forgot to do that. I made the the necessary changes. Actually, since I rebased the code to the latest one, this one is again appearing as a separate commit only (sorry for that).